### ISSUE TITLE
Does ncnn support convert MXNet model with batch normalization been merged into convolution?

### ISSUE THREAD

**Issue by XIANG24** [Tencent/ncnn#703]

**Comment by XIANG24** [comment]
I can convert the MXNet mode successfully. But after merging the batchnorm into convolution layer, it cannot be converted.

**Comment by XIANG24** [comment]
Solved. The format of merged batchnorm net file(json) should be same with original file, as the mxnet2ncnn parses json file by specific format.(Refer to function `read_mxnet_json` in `mxnet2ncnn.cpp`) .

**Comment by nihui** [comment]
The ncnn batch infrastructure feature, which relies on the pnnx tool, is implemented at https://github.com/Tencent/ncnn/commit/127b4146148844ccf67a5bdcb0385e925aa13fbb
